src/test/java/kitchenpos/application/OrderTableServiceTest.java

```diff
@@ -0,0 +1,222 @@
+        // given
+        final OrderTable request = new OrderTable();
+        request.setName("테이블");
+
+        when(orderTableRepository.save(any())).then(invocation -> {
+            OrderTable savedOrderTable = invocation.getArgument(0);
+            savedOrderTable.setId(UUID.randomUUID());
+            return savedOrderTable;
+        });
+
+        // when
+        OrderTable createdOrderTable = orderTableService.create(request);
+
+        // then
+        assertAll(
+            () -> assertNotNull(createdOrderTable),
+            () -> assertNotNull(createdOrderTable.getId()),
+            () -> assertEquals(request.getName(), createdOrderTable.getName()),
+            () -> assertEquals(0, createdOrderTable.getNumberOfGuests()),
+            () -> assertFalse(createdOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("`주문 테이블`을 생성할 때 이름이 없으면 예외가 발생한다.")
+    @Test
+    void createWithEmptyName() {
+        // given
+        final OrderTable orderTable = new OrderTable();
+        orderTable.setName("");
+
+        // when & then
+        assertThatThrownBy(() -> orderTableService.create(orderTable))
+            .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("사용자는 `주문 테이블`에 손님을 앉힐 수 있다.")
+    @Test
+    void sit() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+
+        when(orderTableRepository.findById(orderTable.getId())).thenReturn(
+            Optional.of(orderTable));
+
+        // when
+        OrderTable sittingOrderTable = orderTableService.sit(orderTable.getId());
+
+        // then
+        assertAll(
+            () -> assertNotNull(sittingOrderTable),
+            () -> assertEquals(orderTable.getId(), sittingOrderTable.getId()),
+            () -> assertTrue(sittingOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("사용자는 `주문 테이블`을 비울 수 있다.")
+    @Test
+    void clear() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+        orderTable.setOccupied(true);
+
+        when(orderTableRepository.findById(orderTable.getId())).thenReturn(
+            Optional.of(orderTable));
+        when(orderRepository.existsByOrderTableAndStatusNot(orderTable,
+            OrderStatus.COMPLETED)).thenReturn(false);
+
+        // when
+        OrderTable clearedOrderTable = orderTableService.clear(orderTable.getId());
+
+        // then
+        assertAll(
+            () -> assertNotNull(clearedOrderTable),
+            () -> assertEquals(orderTable.getId(), clearedOrderTable.getId()),
+            () -> assertEquals(0, clearedOrderTable.getNumberOfGuests()),
+            () -> assertFalse(clearedOrderTable.isOccupied())
+        );
+    }
+
+    @DisplayName("`주문 테이블`을 비울 때 주문이 완료되지 않았으면 예외가 발생한다.")
+    @Test
+    void clearWithNotCompletedOrder() {
+        // given
+        final OrderTable orderTable = OrderTableFixture.createOrderTable();
+        orderTable.setOccupied(true);
```

Comment:
fixture 함수릐 메서드 파라미터로 인자들을 주입받아서 fixture를 조금 더 유연하게 생성하도록 만들어 볼 수 없을까요? 🙂

메서드 오버로딩을 잘 활용하여 default 값을 적절히 부여해주면 될 것 같아요. 😃
(꼭 여기뿐만이 아니라 다른 테스트 클래스에서도 동일하게 적용해볼 수 있을 것 같아요) 

```suggestion
        final OrderTable orderTable = OrderTableFixture.createOrderTable(true);
```